app/src/main/java/com/bashkevich/androidfundamentals/model/entity/Movie.kt

```diff
@@ -1,20 +1,19 @@
-package com.bashkevich.androidfundamentals.data
+package com.bashkevich.androidfundamentals.model.entity
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
-import kotlinx.android.parcel.RawValue
 
 @Parcelize
 data class Movie(
     val id: Int,
     val title: String,
     val overview: String,
-    val poster: String,
-    val backdrop: String,
+    val poster: String?,
+    val backdrop: String?,
     val ratings: Float,
     val numberOfRatings: Int,
     val minimumAge: Int,
-    val runtime: Int,
-    val genres: List<Genre>,
-    val actors: List<Actor>
+    val runtime: Int?,
+    val genres: List<Genre>?,
+    var actors: List<Actor>
```

Comment:
Why var?

Reply:
we set list of actors later

Reply:
not when converting from dto

Reply:
It preferable to use immutable objects. Especially when data class generates nice `copy` method